lib/scarpe/para.rb

```diff
@@ -1,13 +1,25 @@
 class Scarpe
   class Para
-    def initialize(app, text)
+    def initialize(app, text, stroke: nil, **html_attributes)
       @app = app
       @text = Array(text)
+      @stroke = stroke
+      @html_attributes = Hash(html_attributes)
       @app.append(render)
     end
 
     def render
-      "<p id=#{object_id}>#{text.join}</p>"
+      HTML.render do |h|
+        h.p(html_attributes.merge(id: object_id, style: style)) do
```

Comment:
⛏️ For clarity what do you think about moving the `html_attributes.merge(id: object_id, style: style)` bit to its own method? Something like `options` or `attributes`. Might just be easier to grok this if it's just
```ruby
h.p(**options) do
  # ...
end
```
Could just be me though